scripts/generate_asm.py

```diff
@@ -0,0 +1,73 @@
+import sys
+import re
+
+def replace_asm(file):
+    """
+    Creates all the possible files with all the combinations of the replacements
+    """
+    with open(file, 'r') as f:
+        lines = f.readlines()
+    replacements = get_replacements(lines)
+    files_to_replace = [lines.copy()]
+    
+    for str_to_replace in replacements:
+        new_files_to_replace = []
+        for _file in files_to_replace:
+            new_files = replace_once(_file,str_to_replace)
+            new_files_to_replace.extend(new_files)
+        if len(new_files_to_replace) != 0:
+            files_to_replace = new_files_to_replace
+
+    for j,_file in enumerate(files_to_replace):
+        with open(file[:-5] + "_replaced_" + str(j) + ".zasm", 'w') as f:
```

Comment:
Check this slice, it's generating these files for me:
```
  exampl_replaced_1.zasm
  exampl_replaced_2.zasm
  exampl_replaced_3.zasm
```

Mind the missing 'e', I was using the example.asm file from the readme.